resources/assets/components/InboxItem/InboxTile.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class InboxTile extends React.Component {
+  displayImage(photo_url) {
```

Comment:
@katiecrane i think you can remove this call in `InboxTile.js` now and just use the helper? I see you updated `index.js` or maybe i missed it.